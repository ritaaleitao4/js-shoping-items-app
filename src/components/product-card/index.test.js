import ProductCard from '.';

const mockItem = {
    modelClass: "A",
    version: "200",
    price: {
        amount: "29500.00",
        currency: "EUR",
        locale: "de-DE"
    },
    imagePath: "/assets/carImages/aclasswhite.jpg"
};

describe('product card component', () => {
    it('it renders correctly', () => {
        expect(new ProductCard(mockItem, [], 0).render()).toMatchSnapshot();
    });
});