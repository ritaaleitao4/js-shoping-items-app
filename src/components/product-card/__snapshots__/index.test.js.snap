// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`product card component it renders correctly 1`] = `
<li class="product-card ">
  <h2>
    A200
  </h2>
  <div class="product-card__content">
    <div class="product-card__content__image">
      <img src="/assets/carImages/aclasswhite.jpg"
           alt="A 200"
      >
    </div>
    <div class="product-card__content__info">
      <p>
        â‚¬29,500.00
      </p>
    </div>
    <button class="add-to-bag"
            id="0"
    >
      Add to Shopping Bag
    </button>
  </div>
</li>
`;
